index.js

```diff
@@ -162,6 +164,87 @@ const inventoryCommand = (message) => {
   });
 };
 
+const lookUpCommand = (message) => {
+  const originalMessage = message.content;
+  const fuzzySearchKey = originalMessage.slice(8);
+
+  axios
+    .get(
+      "https://db.ygoprodeck.com/api/v7/cardinfo.php?&fname=" + fuzzySearchKey
+    )
+    .then(function (cardSearchResults) {
+      console.log(
+        "https://db.ygoprodeck.com/api/v7/cardinfo.php?&fname=" + fuzzySearchKey
+      );
+
+      let searchedCardName;
+      let closestDataIndex = -2;
+      let secondClosestDataIndex;
+      cardSearchResults.data.data.forEach((element, index) => {
+        if (element.name.toLowerCase() === fuzzySearchKey.toLowerCase()) {
+          closestDataIndex = index;
+        } else if (
+          element.name.toLowerCase().startsWith(fuzzySearchKey.toLowerCase())
+        ) {
+          secondClosestDataIndex = index;
+        } else {
+          console.log("This didn't work");
+        }
+      });
+
+      if (closestDataIndex > -1) {
+        const attachment = new Discord.MessageAttachment(
+          cardSearchResults.data.data[closestDataIndex].card_images[0].image_url
```

Comment:
since you use `cardSearchResults.data.data` a lot, might be worth sticking that into a variable